spec/views/collections/index.html.erb_spec.rb

```diff
@@ -8,5 +8,11 @@
   it 'renders a list of collections' do
     render
     expect(rendered).to have_css 'td', text: /My Collection/, count: 2
+    expect(rendered).to have_css 'a', text: /New Workspace/, count: 2
+  end
+
+  it 'renders a links to create workspaces from each collection' do
+    render
+    expect(rendered).to have_css 'a', text: /New Workspace/, count: 2
```

Comment:
I think we may have some other commits in this branch?  Can you rebase these out?  These are going to conflict with changes in #67 (as I had to fix these up after rebasing the PR that introduces these lines).